lib/tokeniser.js

```diff
@@ -53,6 +69,7 @@ const nonRegexTerminals = [
   "Infinity",
   "NaN",
   "Promise",
+  "any",
   "async",
```

Comment:
Again while you are at it, could you remove `async` and `constructor` from this list? They are already in `argumentNameKeywords` so they are redundant.